src/components/Input/index.tsx

```diff
@@ -0,0 +1,42 @@
+interface InputProps {
+  disabled: boolean;
+  label?: string;
+  essential: boolean;
+  limit?: number | undefined;
+  baseData?: string;
+  width: 'small' | 'medium' | 'large';
+}
+
+const Input = ({
+  disabled,
+  label,
+  essential,
+  limit,
+  baseData,
+  width,
```

Comment:
`disabled`, `essential`을 optional로 두고 기본값을 지정해주면 어떨까요?

해당 속성이 필요 없는 경우 Input에 `disabled`, `essential` 속성을 `false`로 전달하는 것보다 아예 적지 않는 것이 사용하기 편리할 것 같아요🙂
```suggestion
  disabled = false,
  label,
  essential = false,
  limit,
  baseData,
  width,
```

Reply:
반영했습니다